apps/sasquatch/src/main/java/com/microsoft/appcenter/sasquatch/activities/SettingsActivity.java

```diff
@@ -494,23 +494,46 @@ public boolean onPreferenceChange(Preference preference, Object newValue) {
                     return true;
                 }
             });
-            initEditText(R.string.app_secret_key, R.string.app_secret_title, APP_SECRET_KEY, getString(R.string.app_secret), new EditTextListener() {
+            final String appSecret = MainActivity.sSharedPreferences.getString(APP_SECRET_KEY, "9e0d97c1-7838-46d0-9dab-1a0ef66aec6e");
```

Comment:
Do not hardcode it: getString(R.string.app_secret)

Reply:
And better, use the first value from the new array resource and remove the old single dimension resource.

Reply:
Thanks, I've fixed this and also removed the now-duplicate "app_secret" resource